astropy/table/tests/test_masked.py

```diff
@@ -195,10 +196,15 @@ def test_incomplete_mask_spec(self):
             MaskedColumn(name="b", length=4, mask=mask_list)
 
 
+STR_DTYPES = ["?", "b", "i2", "f4", "c8", "S", "U", "O"]
+if not NUMPY_LT_2_0:
+    STR_DTYPES.extend([np.dtypes.StrDType, np.dtypes.BytesDType, np.dtypes.StringDType])
```

Comment:
I'm not sure if `np.dtypes.StrDType` and `BytesDType` are necessary since they are apparently exactly the same as `U` and `S`. Having these here is not hurting anything, but at least add a comment like what I just said. 

When I read this the first time I was confused about those two types and started googling, but I had a difficult time discovering any relevant docs. The docstring is not especially helpful either:
```
In [17]: np.dtypes.StrDType?
Init signature: np.dtypes.StrDType(self, /, *args, **kwargs)
Docstring:     
DType class corresponding to the scalar type and dtype of the same name.
```

Reply:
thanks ! indeed, the lack of clear documentation is why I didn't realize this myself. Comments added !